DIPs/DIP1xxx-sw.md

```diff
@@ -0,0 +1,954 @@
+In-place struct initialization
+===============================
+
+| Field           | Value                                                           |
+|-----------------|-----------------------------------------------------------------|
+| DIP:            | N/A
+| RC#             | 0
+| Author:         | Cédric Picard (cpicard@openmailbox.org), Sebastian Wilzbach (seb[at]wilzba[dot]ch)
+| Implementation: | N/A
+| Status:         | N/A
+
+Abstract
+--------
+
+Structs support static initialization. This DIP proposes to extend this
+initialization to in-place struct declaration. This change could notably be
+used to mimic keyword arguments and make argument manipulations at runtime
+easier.
+
+### Links
+
+- [D grammar](https://dlang.org/spec/grammar.html)
+- [Language specs on structs](https://dlang.org/spec/struct.html)
```

Comment:
Maybe change this or add a link to [static struct initialization spec](dlang.org/spec/struct.html#static_struct_init)